website/docs/docs/core/connect-data-platform/fabricspark-setup.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Microsoft Fabric Spark setup"
+description: "Read this guide to learn about the Microsoft Fabric Spark lakehouse setup in dbt."
+id: "fabricspark-setup"
+meta:
+  maintained_by: Microsoft
+  authors: 'Pradeep Srikakolapu'
+  github_repo: 'microsoft/dbt-fabricspark'
+  pypi_package: 'dbt-fabricspark'
+  min_core_version: 'v1.7.0'
+  cloud_support: Not Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'db-fabric-synapse'
+  slack_channel_link: 'https://app.slack.com/client/T0VLPD22H/C01DRQ178LQ'
+  platform_name: 'Fabric Spark'
+  config_page: '/reference/resource-configs/fabricspark-configs'
+---
+
+
+<Snippet path="warehouse-setups-cloud-callout" />
+<Snippet path="dbt-fabricspark-for-fabricspark" />
```

Comment:
hey @prdpsvs - it doesn't look like there's a file for this snippet. is this something you're planning on adding? the deploy is failing bc it can't find fabricspark-for-fabricspark.md.